doc/radosgw/notifications.rst

```diff
@@ -95,20 +95,25 @@ Topics
 
 .. note::
 
-    In all topic actions, the parameters are URL encoded, and sent in the message body using ``application/x-www-form-urlencoded`` content type
+    In all topic actions, the parameters are URL-encoded and sent in the
+    message body using the ``application/x-www-form-urlencoded`` content type.
    
 
 Create a Topic
 ``````````````
 
-This will create a new topic. The topic should be provided with push endpoint parameters that would be used later
-when a notification is created.
-Upon a successful request, the response will include the topic ARN that could be later used to reference this topic in the notification request.
-To update a topic, use the same command used for topic creation, with the topic name of an existing topic and different endpoint values.
+This creates a new topic. Provide the topic with push endpoint parameters,
+which will be used later when a notification is created. Upon a successful
```

Comment:
@zdover23 The sentence beginning with _Upon_ needs to be rewritten at some point.

Reply:
@colemitchell I'll read up on this and figure out a better way to word it.

Reply:
@colemitchell: https://github.com/ceph/ceph/pull/48498 improves this sentence. However, the English throughout notifications.rst is confusing. We should rewrite it. I estimate that the rewriting of notifications.rst (and the insertion of unselectable prompts into the command lines) will take seven to nine hours and seven to ten distinct pull requests.